libs/libc/net/lib_etherntoa_r.c

```diff
@@ -0,0 +1,52 @@
+/****************************************************************************
+ * libs/libc/net/lib_etherntoa_r.c
```

Comment:
Maybe we can follow `libs/libc/net/lib_etheraton.c` example and implement both non-`_r` and `_r` interfaces in a single file?